JS/Lesson34/gulcin/recipe-app/src/App.css

```diff
@@ -1,6 +1,19 @@
+@import url('https://fonts.googleapis.com/css2?family=Titillium+Web:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700&display=swap');
```

Comment:
CSS imports are considered slower that imports in HTML so it's better to prefer HTML imports for fonts. In react application, you'd need to add it to public/index.html file. This is one of the rare cases when you edit files outside `src` directory.

Reply:
Evet bunu önceki ödevden öğrenmiştim, artık public index dosyasına ekliyorum burda böyle kalmış :)